### ISSUE TITLE
Multiple affiliations and multiple authors

### ISSUE THREAD

**Issue by bunni90** [gi-ev/LNI#173]
I have found a bug with multiple affiliations in the latest lni.cls version (1.10).

For a single author with multiple affiliations, the author's data is written under each affiliation (is that intended?). 

If the 1st author has only one affiliation, but the 2nd author has several, it seems to work (the data of the corresponding author(s) is printed under each affiliation).

However, if the 1st author and the 2nd author have several affiliations, the data of the 2nd author is printed under each affiliation as often as authors have specified this affiliation. If both authors have specified the affiliation, the data of the 2nd author will be printed twice; if only the 1st author has specified the affiliation, the name of the 2nd author will still be printed. 

Here is a mwe:
```
\documentclass{lni}

\begin{document}

\title{Title}

\author[1,2,3]{Firstname1 Lastname1}{firstname1.lastname1@affiliation1.org}{0000-0000-0000-0000}
\author[1,2]{Firstname2 Lastname2}{firstname2.lastname2@affiliation2.org}{0000-0000-0000-0000}
\affil[1]{University 1 \\Department\\Address\\Country}
\affil[2]{University 2 \\Department\\Address\\Country}
\affil[3]{University 3 \\Department\\Address\\Country}

\maketitle

\end{document}
```

**Comment by sieversMartin** [comment]
Please try https://github.com/gi-ev/LNI/blob/173-multiple-affiliations-and-multiple-authors/lni.cls

**Comment by bunni90** [comment]
Thank you very much for this quick fix. This reopened the issue of links in the references having a different font. So I just merged your fix from version 1.9 in the version 1.10
[lni.cls.zip](https://github.com/user-attachments/files/16350483/lni.cls.zip)

**Comment by bunni90** [comment]
One issue still remains: That the emails and orcids will be printed under each affiliation.
To deal with that one can stop the script from printing the emails and orcids at all. This can be achieved with various hacks e.g. changing line 377 to `\ProcessList{\@multiaffil}` and then hardcoding the emails and orcids to the first affiliation e.g. 
```
\affil[1]{University 1 \\Department\\Address\\Country\newline firstname1.lastname1@affiliation1.org, 0000-0000-0000-0000, \newline firstname2.lastname2@affiliation2.org, 0000-0000-0000-0000}
```
